priv/site/_includes/css/components/Footer.scss

```diff
@@ -2,102 +2,97 @@
   background-color: var(--c-green);
   color: var(--c-black);
   text-transform: uppercase;
+}
+
+.Footer--alt {
+  background-color: var(--c-purple);
+  color: var(--c-white);
+}
+
+.Footer-contentMobile {
+  padding: var(--s-5) var(--s-4);
+}
+
+.Footer-grid {
+  display: grid;
+  grid-column-gap: var(--s-4);
+  grid-row-gap: var(--s-4);
+  grid-template-columns: repeat(2, 1fr);
+}
 
-  &.Footer--alt {
-    background-color: var(--c-purple);
-    color: var(--c-white);
+.Footer-footer {
+  display: flex;
+  font-size: var(--s-3);
+  font-family: "Chakra Petch";
+  font-weight: 400;
+  justify-content: space-between;
+  line-height: var(--s-4);
+  margin-top: var(--s-6);
+  text-transform: uppercase;
+}
+
+.Footer-followUs {
+  font-weight: 600;
+  text-decoration: underline;
+  text-transform: none;
+}
+
+@media (max-width: 650px) {
+  .Footer-content {
+    display: none;
   }
+}
 
-  .followUs {
-    font-weight: bold;
-    text-decoration: underline;
-    text-transform: none;
-    font-family: input-mono;
+@media (min-width: 650px) {
+  .Footer-contentMobile {
+    display: none;
   }
 
-  .sponsor {
-    font-weight: bold;
-    text-decoration: underline;
-    text-transform: uppercase;
-    font-family: input-mono;
+  .Footer-grid {
+    align-items: flex-start;
+    display: grid;
+    font-family: "input-mono";
+    font-size: var(--s-3);
```

Comment:
If this is a layout component, why is it setting font properties? I would suggest either moving this top, to `Footer` or create a new child that sets the font-properties.